test/new_relic/agent/configuration/yaml_source_test.rb

```diff
@@ -122,6 +122,12 @@ def test_failure_should_include_message
       assert_includes source.failures.flatten.join(' '), 'yolo'
     end
 
+    def test_transaction_threshold_one_liner
```

Comment:
the reason this test is doing things differently is because `transaction_tracer.transaction_threshold` is already in our test newrelic.yml, although it is defined on 2 lines. Since I only changed the way one method works, I thought it would be most straightforward to simply call that method and pass it something different and check how it responds.